src/System.IO/src/System.IO.csproj

```diff
@@ -76,9 +82,14 @@
     <TargetingPackReference Include="System" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'netstandard13aot'">
     <None Include="project.json" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard13aot'">
+    <TargetingPackReference Include="System.Private.Interop" />
+    <None Include="netstandard13aot\project.json" />
```

Comment:
Even better if you don't need to fork it just put this in the main project.json.